content/webapp/services/prismic/events.ts

```diff
@@ -0,0 +1,190 @@
+import { UiEvent, EventTime } from '@weco/common/model/events';
```

Comment:
This is very close to the original code in `events.js`, but properly typed now. 🙌